### PR TITLE
Activate the L shortcut

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

**Comment by tomrf1** [comment]
👍

File: ui/public/javascript/editForm.js [hunk 1]
```diff
@@ -121,6 +121,14 @@ Mousetrap.bind("l", function() {
     document.querySelector(".ingredients-list:last-child").scrollIntoView({
         behavior: 'smooth'
     });
+}
+
+//try to parse a whole block
+Mousetrap.bind("l", function() {
```
Comment (LATaylor-guardian) [inline ui/public/javascript/editForm.js]: Looks like you can assign multiple keys to function, see: https://craig.is/killing/mice

So:

Mousetrap.bind("l", "L", function() {
   parseWholeBlockOfRecipeTextForIngredients()
})

File: ui/public/javascript/editForm.js [hunk 2]
```diff
@@ -103,13 +103,13 @@ Mousetrap.bind("i", function() {
 })
 */
 
-//try to parse a whole block
-Mousetrap.bind("l", function() {
+
+function parseWholeBlockOfRecipeTextForIngredients(){
```
Comment (LATaylor-guardian) [inline ui/public/javascript/editForm.js]: `parseHtmlForIngredients`

**Comment by shtukas** [comment]
@LATaylor-guardian Thank you for the suggestion. That's awesome. I made the update

**Pull request merged**